Loan_Default_Classification_Code_Review_V1.py

```diff
@@ -450,33 +456,13 @@ def histogram_alt(feature):
     #msg = "%s: %f" % (name, roc_auc_score)
 
 
-# In[41]:
-
-
-# prepare configuration for cross validation test harness
-seed = 7
-
-# Compare accuracy score among different models
-results = []
-names = []
-scoring = 'accuracy'
-for name, model in models:
-	kfold = model_selection.KFold(n_splits=10, random_state=seed)
-	cv_results = model_selection.cross_val_score(model, X_train, y_train, cv=kfold, scoring=scoring)
-	results.append(cv_results)
-	names.append(name)
-	msg = "%s: %f (%f)" % (name, cv_results.mean(), cv_results.std())
-	print(msg)
-    
-# boxplot algorithm comparison
-fig = plt.figure()
-fig.suptitle('Algorithm Accuracy Comparison')
-ax = fig.add_subplot(111)
-plt.boxplot(results)
-ax.set_xticklabels(names)
-plt.show()
-
-
+"""
+Interpret this part!
+What is the ROC AUC score?
+Which algorithms are better?
+Why does the performance of some of the algorithms look similar?
+Does this plot indicate other work should be done?
```

Comment:
I am actually not sure here... let me try to answer.
If LR and LDA performance look very identical, does that mean I should do some parameter tuning to decide which one is EVEN better? I read about the link I provided previously, the author mentioned about the A, B ,C, Q indexes... but it is more on stats approach. Is that where I should head towards?